#### Genetic associations

Genetic associations are available for both common and rare \(Mendelian\) diseases and come from the following sources:

Common:

[GWAS catalog](/data-sources/genetic-associations/gwas-catalog.md)

Rare:

* * [Genomics England PanelApp](http://www.targetvalidation.org/#genomics_england)
  * [PheWAS catalog](http://www.targetvalidation.org/#phewas)
  * [UniProt](http://www.targetvalidation.org/#uniprot)
  * [UniProt literature](http://www.targetvalidation.org/#uniprot_lit)
  * [European Variation Archive \(EVA\)](http://www.targetvalidation.org/#eva)
  * [Gene2Phenotype](http://www.targetvalidation.org/#gene2phenotype)

##### 

##### Genomics England PanelApp

The[Genomics England PanelApp](https://panelapp.extge.co.uk/crowdsourcing/PanelApp/)is a knowledgebase that combines crowdsourcing of expertise with curation to provide gene-disease relationships to aid the clinical interpretation of genomes within the 100,000 Genomes Project. This information is curated initially from four sources: Radboud University Medical Center, Illumina TruGenome Predisposition Screen, Emory Genetic Laboratory, and the UK Genetic Testing Network. The genes curated from at least three of these sources are initially classified as ‘green genes’. Additional sources may be used for the curation of some of the phenotypes, genes or mode of inheritance \(e.g. peer-reviewed published articles, OMIM, expert list\). Expert review of the gene panels for a particular disease is then sought in PanelApp, and each panel is revised. Only green diagnostic-grade genes on version 1+ panels \(which have the highest level of confidence\) are utilised for genome analysis within the 100,000 Genomes Project. We include in our Open Targets Platform the ‘green genes’ from version 1+ panels along with their phenotypes, providing the latter can be mapped to a disease or phenotype ontology.

##### PheWAS catalog

The[PheWAS](https://phewascatalog.org/)\(phenome-wide association studies\) resources provide associations between a genetic variant and multiple phenotypes. It contains clinical phenotypes derived from the electronic medical record \(EMR\)-linked DNA biobank BioVU by the Center for Precision Medicine at the Vanderbilt University Medical Center. The EMR-based PheWAS uses ICD9 \(International Classification of Disease, 9th edition\), which were mapped to EFO usingOLS andZooma. The catalog contains all associations with p &lt; 0.05 \(uncorrected\).

